Tutorial split (#145)
pretix/pretix-docs: docs/guides/event-series.md

 - a theater performing different plays multiple times in the same venue
 - a touring act playing the same concert in several different locations
 - workshops with limited capacity hosted several times
 - an escape room allowing customers to book a time slot
 - a museum offering guided tours

There are some situations in which an event series is not the most straightforward solution.
We recommend creating **singular events** for the following use cases:

 - events that take place over the course of several days, such as conferences or festivals, if customers typically attend more than one day of the event.
 This can be solved more easily using different products and check-in rules.
 - events with large gaps of time between the individual dates, such as annual conferences
 - events with very little overlap in terms of organizational parameters

### How to create an event series

Navigate to the dashboard.
In the list titled "Your upcoming events", click the :btn-icon:fa3-plus:Create a new event: button.
Under "Event type", select "Event series or time slot booking".
The rest of the process is the same as the creation of a singular event.

> For more information on creating an event, see the tutorial page on [event creation](../tutorial/single-events/event.md).

Reviewer: > For more information on creating an event, see the tutorial page on [event creation](../tutorial/single-events/event.md).

Ergibt das Sinn, an dieser Stelle auf single-events zu verlinken?

### Creating and editing dates in the event series

Navigate to :navpath:Your event series → :fa3-calendar: Dates:.
This page displays the list of currently existing event dates and allows you to create, edit, clone and delete them.

![Page titled "Dates", displaying buttons for creating a single or multiple new dates. ](../assets/screens/event-series/dates.png "Dates screenshot" )

In order to create a single new date for this event series, click :btn-icon:fa3-plus: Create a new date:.
Choose a name, an event start time, and at least one product for the date.
Once you are happy with your choices for the mandatory settings and the many optional settings on the page, click the :btn:Save: button.
You will be taken back to the dates overview page titled "Dates".
The date you just created is now displayed in the list on this page.

In order to create several new dates for this event series, click :btn-icon:fa3-plus: Create many new dates:.
The process is very similar to creating a single date.

> It is illustrated in the example below.

Reviewer: ```suggestion
It is illustrated in the following example.
```

Nicht unbedingt schöner, aber "below" schafft zumindest bei mir eine andere Erwartung als "das ist der nächste Satz".

![Page titled "Create multiple dates", displaying options for a repetition rule and times, as well as a preview of dates about to be created. ](../assets/screens/event-series/create-multiple-example.png "Create multiple dates example" )

As a practical example, let us assume you are offering an event titled "Sales workshop" every Monday and Wednesday at 10 AM in the second half of the year 2025.
In the box titled "Repetition rule", you would make the following choice:
"Repeat every 1 week(s), starting at 2025-07-01."
Check the boxes next to the day of the week when the dates take place, "Monday" and "Wednesday" for example.
Select the radio button next to "Repeat until" and choose a date such as 2025-12-31.

The "Preview" on the right side of the page is useful for double-checking the repetition rule.
It will display the first ten and last ten dates you are about to create.

Under "Times", click the :btn-icon:fa3-plus: Add a single time slot: button.
Choose the start time for your event, for example, 10:00:00.
Optionally, you may choose an event end time such as 12:00:00 and an admission time such as 09:30:00.

> Under "General information", enter a name descriptive name for these dates in the "Name" field, "Sales workshop" for example.

Reviewer: ```suggestion
Under "General information", enter a descriptive name for these dates in the "Name" field, "Sales workshop" for example.
```

Under "Quotas", choose the product that shall grant admission to these dates in the "Products" field.

Once you have taken these steps and made your optional choices, click the :btn:Save: button.
You will be taken back to the dates overview page titled "Dates".

> If your venue closes on Christmas Eve and New Year's Eve, check the boxes next to those dates, scroll to the bottom of the page and click the :btn-icon:fa3-trash: Delete selected (2): button.

Reviewer: > If your venue closes on Christmas Eve and New Year's Eve, check the boxes next to those dates, scroll to the bottom of the page and click the :btn-icon:fa3-trash: Delete selected (2): button.

Das ist ebenso deutschlandzentrisch wie ungewöhnlich, können wir entweder ein gängigeres oder ein ungewöhnlicheres Beispiel finden?

## Troubleshooting

### The button :fa3-calendar: Dates: does not appear in the sidebar on the left

> If the button :btn-icon:fa3-calendar: Dates: does not appear in the sidebar on the left, there are three possible explanations for this:

Reviewer: > If the button :btn-icon:fa3-calendar: Dates: does not appear in the sidebar on the left, there are three possible explanations for this:

Ist hier hinreichend klar, was für ein "sidebar on the left" gemeint sein will, wenn Lesende direkt in den Abschnitt Troubleshooting springen?

Choose the start time for your event, for example, 10:00:00.
Optionally, you may choose an event end time such as 12:00:00 and an admission time such as 09:30:00.

Under "General information", enter a descriptive name for these dates in the "Name" field, for example: `Sales workshop`.
Under "Quotas", choose the product that shall grant admission to these dates in the "Products" field.

Once you have taken these steps and made your optional choices, click the :btn:Save: button.
You will be taken back to the dates overview page titled "Dates".

> If your venue has to close for a few of those dates (due to maintenance, team building, or any other reason), check the boxes next to those dates, scroll to the bottom of the page and click the :btn-icon:fa3-trash: Delete selected (2): button.

Reviewer: ```suggestion
If your venue has to close for a few of those dates (due to maintenance, internal team events, or any other reason), check the boxes next to those dates, scroll to the bottom of the page and click the :btn-icon:fa3-trash: Delete selected (2): button.
```
